### PR TITLE
add cpr

### PR DESCRIPTION
**Pull request by dvirtz**

### CODE REVIEW

File: cmake/configs/default.cmake [hunk 1]
```diff
@@ -347,3 +347,4 @@ hunter_config(zookeeper VERSION 3.4.9-p2)
 hunter_config(tacopie VERSION 2.4.0-h1)
 hunter_config(cpp_redis VERSION 3.5.0-h1)
 hunter_config(IF97 VERSION 2.1.2)
+hunter_config(cpr VERSION 1.3.0)
```
Comment (ruslo) [inline cmake/configs/default.cmake]: Try to keep it alphabetical.

File: cmake/projects/cpr/hunter.cmake [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+)
+
+hunter_cmake_args(cpr CMAKE_ARGS 
+  BUILD_CPR_TESTS=OFF
+  CMAKE_USE_OPENSSL=OFF
+)  
+
+hunter_pick_scheme(DEFAULT url_sha1_cmake)
+hunter_cacheable(cpr)
+hunter_download(PACKAGE_NAME cpr PACKAGE_INTERNAL_DEPS_ID "4" )
```
Comment (ruslo) [inline cmake/projects/cpr/hunter.cmake]: It's standard CMake-based scheme, no need to use `PACKAGE_INTERNAL_DEPS_ID`.

File: docs/packages/pkg/cpr.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.. spelling::
+
+    cpr
+
+.. index:: unsorted ; cpr
```
Comment (ruslo) [inline docs/packages/pkg/cpr.rst]: `s,unsorted,networking,`?

File: docs/packages/pkg/cpr.rst [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+.. _pkg.cpr:
+
+cpr
+============
```
Comment (ruslo) [inline docs/packages/pkg/cpr.rst]: We need only 3:
```
cpr
===
```

File: cmake/configs/default.cmake (hunk 1 above)
Comment (dvirtz) [reply cmake/configs/default.cmake]: I think this ship has long sailed

Comment (ruslo) [reply cmake/configs/default.cmake]: Too bad there is no automatic check. Though I believe it's possible to craft one :)

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2832
* https://travis-ci.org/ingenue/hunter/builds/303428945?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2833
* https://travis-ci.org/ingenue/hunter/builds/303537010?utm_source=github_status&utm_medium=notification

**Comment by dvirtz** [comment]
Thanks.

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.160

**Pull request merged**